### ISSUE TITLE
tensorflow package for python3 not found in tensorflow-1.14.0-notebook-gpu:0.7.0

### ISSUE THREAD

**Issue by Dibel** [kubeflow/kubeflow#4471]
/kind bug

**What steps did you take and what happened:**

Try to set up a notebook with image gcr.io/kubeflow-images-public/tensorflow-1.14.0-notebook-gpu:v0.7.0 in kubeflow 0.7. But this image contains both python2 and python3 runtime, and tensorflow only exists in python2 runtime.

In container's terminal I run `python --version` got the following output:
`Python 2.7.15+`

`pip list` got the following output:
<details>
  <summary>Click to expand</summary>
  
```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Package                            Version
---------------------------------- -----------
absl-py                            0.7.1
asn1crypto                         0.24.0
astor                              0.8.0
attrs                              19.1.0
backports-abc                      0.5
backports.functools-lru-cache      1.5
backports.shutil-get-terminal-size 1.0.0
backports.weakref                  1.0.post1
bleach                             3.1.0
configparser                       3.7.4
crcmod                             1.7
cryptography                       2.1.4
cycler                             0.10.0
decorator                          4.4.0
defusedxml                         0.6.0
entrypoints                        0.3
enum34                             1.1.6
funcsigs                           1.0.2
functools32                        3.2.3.post2
futures                            3.2.0
gast                               0.2.2
google-pasta                       0.1.7
grpcio                             1.21.1
h5py                               2.9.0
idna                               2.6
ipaddress                          1.0.17
ipykernel                          4.10.0
ipython                            5.8.0
ipython-genutils                   0.2.0
ipywidgets                         7.4.2
Jinja2                             2.10.1
jsonschema                         3.0.1
jupyter                            1.0.0
jupyter-client                     5.2.4
jupyter-console                    5.2.0
jupyter-core                       4.5.0
jupyter-http-over-ws               0.0.6
Keras-Applications                 1.0.8
Keras-Preprocessing                1.1.0
keyring                            10.6.0
keyrings.alt                       3.0
kiwisolver                         1.1.0
Markdown                           3.1.1
MarkupSafe                         1.1.1
matplotlib                         2.2.4
mistune                            0.8.4
mock                               3.0.5
nbconvert                          5.5.0
nbformat                           4.4.0
notebook                           5.7.8
numpy                              1.16.4
pandocfilters                      1.4.2
pathlib2                           2.3.3
pexpect                            4.7.0
pickleshare                        0.7.5
pip                                19.1.1
prometheus-client                  0.7.1
prompt-toolkit                     1.0.16
protobuf                           3.8.0
ptyprocess                         0.6.0
pycrypto                           2.6.1
Pygments                           2.4.2
pygobject                          3.26.1
pyparsing                          2.4.0
pyrsistent                         0.15.2
python-dateutil                    2.8.0
pytz                               2019.1
pyxdg                              0.25
pyzmq                              18.0.1
qtconsole                          4.5.1
scandir                            1.10.0
SecretStorage                      2.3.1
Send2Trash                         1.5.0
setuptools                         41.0.1
simplegeneric                      0.8.1
singledispatch                     3.4.0.3
six                                1.11.0
subprocess32                       3.5.4
tensorboard                        1.14.0
tensorflow-estimator               1.14.0
tensorflow-gpu                     1.14.0
termcolor                          1.1.0
terminado                          0.8.2
testpath                           0.4.2
tornado                            5.1.1
traitlets                          4.3.2
wcwidth                            0.1.7
webencodings                       0.5.1
Werkzeug                           0.15.4
wheel                              0.30.0
widgetsnbextension                 3.4.2
wrapt                              1.11.2
WARNING: You are using pip version 19.1.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```
</details>

`pip3 list` got the following output:
<details>
  <summary>Click to expand</summary>
  
```
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
argo-models (2.2.1a0)
asn1crypto (0.24.0)
attrs (19.1.0)
backcall (0.1.0)
bleach (3.1.0)
cachetools (3.1.1)
certifi (2019.9.11)
cffi (1.12.3)
chardet (3.0.4)
Click (7.0)
cloudpickle (1.2.2)
cryptography (2.7)
decorator (4.4.0)
defusedxml (0.6.0)
docker (4.0.2)
entrypoints (0.3)
fairing (0.5)
future (0.17.1)
google-api-core (1.14.2)
google-api-python-client (1.7.11)
google-auth (1.6.3)
google-auth-httplib2 (0.0.3)
google-cloud-core (1.0.3)
google-cloud-storage (1.20.0)
google-resumable-media (0.4.1)
googleapis-common-protos (1.6.0)
httplib2 (0.14.0)
idna (2.8)
ipykernel (5.1.2)
ipython (7.8.0)
ipython-genutils (0.2.0)
jedi (0.15.1)
Jinja2 (2.10.1)
json5 (0.8.5)
jsonschema (3.0.2)
jupyter-client (5.3.3)
jupyter-console (6.0.0)
jupyter-core (4.5.0)
jupyterlab (1.1.4)
jupyterlab-server (1.0.6)
kfp (0.1.22)
kfp-server-api (0.1.18.3)
kubernetes (10.0.1)
MarkupSafe (1.1.1)
mistune (0.8.4)
nbconvert (5.6.0)
nbformat (4.4.0)
notebook (6.0.1)
numpy (1.17.2)
oauth2client (4.1.3)
oauthlib (3.1.0)
pandocfilters (1.4.2)
parso (0.5.1)
pexpect (4.7.0)
pickleshare (0.7.5)
pip (9.0.1)
prometheus-client (0.7.1)
prompt-toolkit (2.0.9)
protobuf (3.9.2)
ptyprocess (0.6.0)
pyasn1 (0.4.7)
pyasn1-modules (0.2.6)
pycparser (2.19)
Pygments (2.4.2)
pygobject (3.26.1)
PyJWT (1.7.1)
pyrsistent (0.15.4)
python-apt (1.6.4)
python-dateutil (2.8.0)
pytz (2019.2)
PyYAML (5.1.2)
pyzmq (18.1.0)
requests (2.22.0)
requests-oauthlib (1.2.0)
requests-toolbelt (0.9.1)
rsa (4.0)
scipy (1.3.1)
Send2Trash (1.5.0)
setuptools (39.0.1)
six (1.12.0)
tabulate (0.8.3)
terminado (0.8.2)
testpath (0.4.2)
tornado (5.1.1)
traitlets (4.3.2)
uritemplate (3.0.0)
urllib3 (1.24.3)
wcwidth (0.1.7)
webencodings (0.5.1)
websocket-client (0.56.0)
xgboost (0.90)
```
</details>

**What did you expect to happen:**

Only python3 runtime exists in the image, and tensorflow should be useable in python3.

**Anything else you would like to add:**

Everything worked well in tensorflow-1.15.0-notebook-gpu:v-base-e60ee44-1186280489095270404

**Environment:**

- Kubeflow version: 0.7.0
- kfctl version:  v0.7.0
- Kubernetes platform:  bare metal
- Kubernetes version: 
```Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T11:05:50Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T11:05:50Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```
- OS: CentOS Linux 7 (Core)

**Comment by Tabrizian** [comment]
@Dibel Thanks for reporting this problem. We are working on it. Sorry for the inconvenience.

**Comment by jlewi** [comment]
After #4478 is merged we will need to build a new image and move the 0.7.0 tag.

**Comment by jlewi** [comment]
@Tabrizian do we have an updated jupyter image that we can tag?

gcr.io/kubeflow-images-public/tensorflow-1.14.0-notebook-gpu:v0.7.0 is currently pointing to 
sha256:895aa500ff09c9080e0755044e4fd9a3bd3dc3ae403415e220ebcc1dab09b6ef

Looks like it was built on october 10.
v-base-ef41372-1177829795472347138 

There's a newer image
gcr.io/kubeflow-images-public/tensorflow-1.14.0-notebook-gpu@sha256:0d4f11de65e23deac6c2e955b37aa5d549085f8ea1a03d696a3fff9f27d4df3f
	
v-base-b813c9a-1184585142199390209

**Comment by jlewi** [comment]
Looks like this image isn't new enough

```
git log -1 --format='%ci' b813c9a
2019-10-16 14:40:41 -0700
git log -1 --format='%ci' 884a0c51
2019-11-07 06:19:09 -0800
```

**Comment by Tabrizian** [comment]
@jlewi The `1.14.0` has not been updated but I created a pull request for `1.15.0`.

The pull request is here: https://github.com/kubeflow/kubeflow/pull/4352

**Comment by jlewi** [comment]
Closing this as fixed by #4352
